action.yml

```diff
@@ -203,6 +202,7 @@ runs:
         [[ $ANSIBLE_VERBOSE == "true" ]] && command="$command --ansible-verbose"
         [[ -n $PROTOCOL_VERSION ]] && command="$command --protocol-version $PROTOCOL_VERSION"
         [[ $PUBLIC_RPC == "true" ]] && command="$command --public-rpc"
+        [[ -n $FAUCET_VERSION ]] && command="$command --faucet-version $FAUCET_VERSION"
         [[ -n $SAFE_VERSION ]] && command="$command --safe-version $SAFE_VERSION"
```

Comment:
The `deploy` command no longer has the `--safe-version` argument, so we should probably remove that to avoid confusion?